server/src/controllers/auth.controller.ts

```diff
@@ -1,24 +1,23 @@
 import { NextFunction, Request, Response } from "express";
 import { StatusCodes } from "http-status-codes";
+import dotenv from "dotenv";
+
+import { TVerificationsShema, IRequest } from "../types/index";
```

Comment:
`nit` 중요하지는 않지만, 오탈자가 있습니다.

```suggestion
import { TVerificationsSchema, IRequest } from "../types/index";
```